torch/autograd/grad_mode.py

```diff
@@ -104,6 +104,10 @@ class no_grad(_DecoratorContextManager):
         disable gradients locally see :ref:`locally-disable-grad-doc` for
         more information on how they compare.
 
+    .. note::
+        Currently this API does not apply to :ref:`forward-mode AD <forward-mode-ad>`
+        (subject to change). As a workaround you can unpack your dual tensors.
```

Comment:
> Still subject to change? I don't think we will ever change that behavior silently. If we really want to add a new one, it will be via another context manager or by adding new flag to this one.

Ah makes sense. I didn't realize "subject to change" connotes that it changes silently.